package.json

```diff
@@ -10,14 +10,14 @@
     "copy-assets": "gulp sass copy-files",
     "setup": "yarn compile && yarn copy-assets",
     "start": "node src/main/server.js",
-    "start-dev": "./node_modules/.bin/ts-node src/main/server.ts",
+    "start-dev": "TS_NODE_FAST=true ./node_modules/.bin/ts-node src/main/server.ts",
```

Comment:
not planning on upgrading back to 4.0?

Reply:
Not now due to the fact that event if we did not have an issue with nodemon compilation in ts-node 4.0 it is still slower. I will raise issues with them and let them fix it.